database/DataGenerationScripts/generateCSV/generateProducts1CSV.js

```diff
@@ -0,0 +1,69 @@
+var fs = require('fs');
+const faker = require('faker');
```

Comment:
from what i've been told, faker slows down the datagenerating process, so instead of using faker, you could also use pre-generated data and use that instead of faker. But it's up to you if you think your generating speed is fast enough

Reply:
also, your json generators don't have any faker inside, maybe you can use the same format for your csv generators